.gitignore

```diff
@@ -1,3 +1,6 @@
+# IDE
+.idea/
```

Comment:
Usally I ask people put their own IDE stuff in their global gitignore, saves on updating this file constantly

Reply:
Ah ok, I don't remove this.
However, some projects include some shared IDE settings from this directory, so I can't put it in my global